glide.lock

```diff
@@ -31,4 +33,8 @@ imports:
   version: 7ce08ca145dbe0e66a127c447b80ee7914f3e4f9
   subpackages:
   - errorutil
+- name: golang.org/x/net
+  version: d379faa25cbdc04d653984913a2ceb43b0bc46d7
+  subpackages:
+  - context
```

Comment:
@cyphar `context` was added in Go 1.7. If we export interfaces that use `golang.org/x/net/context`, we will be stuck using that package forever.